src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -1009,6 +1010,10 @@ public object ToScriptSource(int executionContextId, object executionContextAuxD
                 executionContextId,
                 executionContextAuxData,
                 //hash:  should be the v8 hash algo, managed implementation is pending
+                isLiveEdit = false,
+                isModule = false,
+                scriptLanguage = "JavaScript",
+                hasSourceURL = false,
```

Comment:
Where are these from?

Reply:
This was needed to implement continueWith, I'll remove this from this PR.